charts/fluentd/Chart.yaml

```diff
@@ -2,5 +2,17 @@ apiVersion: v2
 name: fluentd
 description: A Helm chart for Kubernetes
 # type: application
-version: 0.1.2
+version: 0.1.3
 appVersion: v1
+icon: https://www.fluentd.org/assets/img/miscellany/fluentd-logo_2x.png
+home: https://www.fluentd.org/
+sources:
+  - https://github.com/fluent/fluentd/
+  - https://github.com/fluent/fluentd-kubernetes-daemonset
+maintainers:
+  - name: edsiper
+    email: eduardo@treasure-data.com
+  - name: naseemkullah
+    email: naseem@transit.app
+  - name: Towmeykaw
+    email: towmeykaw@gmail.com
```

Comment:
@Towmeykaw and @edsiper are you ok with maintaining this?

If not we can remove them @sppwf and just ensure end of file has an empty line so that the lint check passes.

Reply:
Yeah you can remove me, I have never used fluentd so wont be much help 👍

Reply:
I also think I will not be a long term maintainer of this one since I am only using fluent-bit in production today.

@sppwf would you like to be a maintainer? If so please add yourself.

@Miouge1 would you consider using this chart instead of the one in helm repository? It's a different image but we would like to achieve same feature parity. If so would be great to have you as maintainer as well?